activiti-cloud-runtime-bundle-service/activiti-cloud-starter-runtime-bundle/src/test/java/org/activiti/cloud/starter/tests/services/audit/AuditProducerIT.java

```diff
@@ -224,8 +224,9 @@ public void shouldProduceEventsDuringSimpleProcessExecution() {
                 .extracting(cloudRuntimeEvent -> ((ProcessInstance) cloudRuntimeEvent.getEntity()).getBusinessKey())
                 .containsExactly("my business key");
             assertThat(receivedEvents).filteredOn(cloudRuntimeEvent -> PROCESS_STARTED.equals(cloudRuntimeEvent.getEventType()))
-                .extracting(cloudRuntimeEvent -> ((ProcessInstance) cloudRuntimeEvent.getEntity()).getName())
-                .containsExactly("my instance name");
+                .extracting(cloudRuntimeEvent -> ((ProcessInstance) cloudRuntimeEvent.getEntity()).getName(),
```

Comment:
In addition to this test that is covering the producer side (runtime-bundle). You also need to add an integration test covering the consumer side (query-service). I think this could be achieved by modifying one of the existing tests in [QueryProcessInstancesEntityIT](https://github.com/Activiti/activiti-cloud/blob/develop/activiti-cloud-query-service/activiti-cloud-starter-query/src/test/java/org/activiti/cloud/starter/tests/QueryProcessInstancesEntityIT.java), maybe [this one](https://github.com/Activiti/activiti-cloud/blob/develop/activiti-cloud-query-service/activiti-cloud-starter-query/src/test/java/org/activiti/cloud/starter/tests/QueryProcessInstancesEntityIT.java#L231).

Reply:
Thank you for the information. I' ve added some check in the test.